packages/services/f-statistics/src/defaultOptions.js

```diff
@@ -0,0 +1,6 @@
+const defaultConfig = {
```

Comment:
Wouid it be better to throw instead of providing defaults?

Reply:
Would rather provide defaults in this case because I would rather capture some data without the perfect configuration, than nothing at all; making it error would be problematic as it would error during interception which could cause the XHR to fail - this is more of a nice to have at this stage.